scipy/stats/_discrete_distns.py

```diff
@@ -152,6 +153,80 @@ def _entropy(self, p):
 bernoulli = bernoulli_gen(b=1, name='bernoulli')
 
 
+class betabinom_gen(rv_discrete):
+    r"""A beta-binomial discrete random variable.
+
+    %(before_notes)s
+
+    Notes
+    -----
+    The probability mass function for `betabinom` is:
+
+    .. math::
+
+       f(k) = \left(\begin{array}n \\ k\end{array}\right)
+            \frac{B(k + a, n - k + b)}{B(a, b)}
+
+    for ``k`` in ``{0, 1,..., n}``, where :math:`B(a, b)` is the beta function.
+
+    `betabinom` takes :math:`n`, :math:`a`, and :math:`b` as shape parameters.
+
+    References
+    ----------
+    .. [1] https://en.wikipedia.org/wiki/Beta-binomial_distribution
+
+    %(after_notes)s
+
+    .. versionadded:: 1.4.0
+
+    %(example)s
+
+    """
+
+    def _rvs(self, n, a, b):
+        p = self._random_state.beta(a, b, self._size)
+        return self._random_state.binomial(n, p, self._size)
+
+    def _get_support(self, n, a, b):
+        return 0, n
+
+    def _argcheck(self, n, a, b):
+        return (n >= 0) & beta._argcheck(a, b)
+
+    def _logpmf(self, x, n, a, b):
+        k = floor(x)
+        combiln = gamln(n + 1) - (gamln(k + 1) + gamln(n - k + 1))
+        return combiln + betaln(k + a, n - k + b) - betaln(a, b)
+
+    def _pmf(self, x, n, a, b):
+        return exp(self._logpmf(x, n, a, b))
+
+    def _stats(self, n, a, b, moments = 'mv'):
```

Comment:
Pep8 nit: no spaces around = for default parameters.

Reply:
Fixed.